src/visualizers-workbench/P5VisualizerTemplate.ts

```diff
@@ -255,5 +255,6 @@ because infinity is, well, infinitely far away!
 // export module constructor
 export const exportModule = new VisualizerExportModule(
     P5VisualizerTemplate,
-    'Step through entries one at a time'
+    P5VisualizerTemplate.prototype.name,
+    P5VisualizerTemplate.prototype.description
```

Comment:
It is good that you have straightened out name and description per #326, but if they are always going to be extracted from the visualizer class in this way, it should be done in the VisualizerExportModule() function, not repeatedly in the arguments to each call thereof. Thanks.

Reply:
You've removed the redundancy here, excellent! Just one tiny remaining point: the comment just above this is now outdated/incorrect: it says to put the class and description into the export module constructor, but it should be up above in the class itself (which is the case and is better, just the comment is now misleading). Thanks!